client/disputes/utils.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { CachedDispute, EvidenceDetails } from 'wcpay/types/disputes';
+
+interface IsDueWithinProps {
+	dueBy?: CachedDispute[ 'due_by' ] | EvidenceDetails[ 'due_by' ];
```

Comment:
I don't see any `isDueWithin()` call that does not pass `dueBy`.

```suggestion
	dueBy: CachedDispute[ 'due_by' ] | EvidenceDetails[ 'due_by' ];
```

Reply:
Thanks!